src/constants/colors/index.js

```diff
@@ -132,10 +151,16 @@ export const guideColors = {
   recommendedCleansersGuide: '#cad1b5',
 };
 
-export default Object.assign(
-  {},
-  brandColors,
-  colorAliases,
-  postcardColors,
-  guideColors
-);
+const colorsCompilation = {
+  ...brandColors,
+  ...colorAliases,
+  ...postcardColors,
+  ...guideColors,
+};
+
+const deprecatedProperties = {
+  purple80: 'purple80 will be deprecated in v2. Use purple85 instead',
```

Comment:
It should probably say `COLORS.purple80 will be deprecated in v3. Use COLORS.purple85 instead.`

Reply:
I understand deprecation as discouragement to use, So we are effectively deprecating now in v2 and remove in v3. or we will deprecate in v3 and remove in v4?

Reply:
we can deprecate now (v2) and remove in v3